### PR TITLE
Telegraf now run under upstart

### PR DESCRIPTION
**Pull request by amitbose327**
I updated telegraf on all the influx hosts to run now under Upstart. 
Deleted the old telegraf file : /var/log/telegraf/telegraf.log
After I confirmed, that telegraf connected with Influxdb correctly, I turned off the console to "none" (This is to avoid the parse errors happening from Moz-local metrics)

@ankanm and @jordmoz Please review

### CODE REVIEW

File: ansible/roles/monitoring-agent/files/collectd-plugins/marathon [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 11a7277c12174e02e68c2cc322b33768515d88dd
+Subproject commit 96fe35d3bf36f4bc5200f60f71d04de95e090f6c
```
Comment (jordmoz) [inline ansible/roles/monitoring-agent/files/collectd-plugins/marathon]: I don't think that's right.  Seems like you're reverting the marathon plugin to the previous, non working version.

File: ansible/roles/monitoring-statsd-fe-influxdb/tasks/main.yml [hunk 2]
```diff
@@ -34,4 +34,21 @@
...
+- name: Remove init.d script for telgraf
+  file: path=/etc/init.d/telegraf state=absent
+  sudo: yes
+  tags:
+    - monitoring
+    - statsd
+    - statsd-fe-influxdb
+
+- name: Install upstart script for telegraf
+  template: src=telegraf.conf.init.j2 dest=/etc/init/telegraf.conf
```
Comment (jordmoz) [inline ansible/roles/monitoring-statsd-fe-influxdb/tasks/main.yml:46]: conf.init -> conf.upstart ?

File: ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.init.j2 [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+limit nofile 65536 65536
+
+start on (net-device-up and local-filesystems and runlevel [2345])
+stop on runlevel [!2345]
+
+respawn
+respawn limit 10 5
+
+console none
+setuid telegraf
```
Comment (jordmoz) [inline ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.init.j2:12]: does this user already exist? does the install create it?

**Comment by jordmoz** [comment]
LGTM, apart from that submodule update.  Please do not commit that.

File: ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.init.j2 (hunk 3 above)
Comment (amitbose327) [inline ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.init.j2:12]: The install creates it.

**Pull request closed without merging**